scripts/buildandrun.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
+set -e
+
+go build ./cmd/spirit
+
+if [ -n "$REPLICA_DSN" ]; then
+  ./spirit --replica-dsn "$REPLICA_DSN" --host $HOST --username $USERNAME --password $PASSWORD --database=$DATABASE --table=$TABLE
```

Comment:
Can we consider supporting `MYSQL_DSN` and `REPLICA_DSN` environment variables as overrides to the spirit CLI?

Reply:
I'd prefer not having spirit behave specially in the presence of bizarre environment variables when we have command-line options that do the job just fine.